app/models/group.rb

```diff
@@ -12,7 +12,9 @@ class Group < ApplicationRecord
   belongs_to :draw
   has_one :suite
   has_many :memberships, dependent: :delete_all
-  has_many :members, through: :memberships, source: :user
+  has_many :full_memberships, -> { where(status: 'accepted') },
+           class_name: 'Membership', inverse_of: :group
```

Comment:
This is tested and still passed, so I'm going to say it's not necessary: https://github.com/YaleSTC/vesta/blob/159_leaders_are_safe/spec/models/group_spec.rb#L65

Feel free to merge if there was nothing else!